app/graphql/types/query_type.rb

```diff
@@ -2,6 +2,9 @@
 
 module Types
   class QueryType < Types::BaseObject
+
+    field :characters, [Types::CharacterType], null: false
```

Comment:
Nice job on all this and getting circle ci to finally pass!  We may need to come and revisit this and make the same query names as the FE to talk to.